packages/tasit-action/src/Contract.js

```diff
@@ -123,11 +133,15 @@ export class Contract extends Subscription {
 
       const tx = this.#ethersContract[f.name].apply(null, args);
 
-      const subscription = new TransactionSubscription(
-        tx,
-        this.#provider,
-        this
-      );
+      const subscription = new TransactionSubscription(tx, this.#provider);
+
+      const errorListener = message => {
+        const { error } = message;
+        this._emitErrorEvent(new Error(`${error.message}`));
+      };
+
+      subscription.on("error", errorListener);
```

Comment:
The diff would have been more easily readable if we named subscriptions as either `actionSub` or `contractEventsSub` (I end up needed to expand unchanged code to find where I am in the file otherwise). Thanks!

Reply:
Done

Reply:
Great, thanks. Resolving this.